src/components/common/Chip/Chip.tsx

```diff
@@ -0,0 +1,13 @@
+interface ChipProps {
+  label: string;
+  className: string;
+  handleClick: React.MouseEventHandler<HTMLDivElement>;
+}
+
+export const Chip = ({ label, className, handleClick }: ChipProps) => {
+  return (
+    <div className={className} onClick={handleClick}>
+      {label}
+    </div>
+  );
+};
```

Comment:
```suggestion
interface ChipProps extends HTMLAttributes<HTMLDivElemtents> {
  label: string;
}

function Chip({ label, ...rest }: ChipProps) {
  return (
    <div {...rest}>
      {label}
    </div>
  );
 };
```
`HTMLAttributes`, `HTMLElements` 타입을 활용하면 핸들러 함수 일일이 추가 안해줘도 유연하게 prop들을 전달할 수 있어!!

참고로 난 on과 handle의 구분을 다음과 같이 하는데 이 네이밍은 같이 정해보자!
1. on : DOM 노드와 유사한 사용성을 위해 prop들 이름으로 사용
2. handle: 컴포넌트 내부에서 커스텀 함수를 네이밍 할 때 사용

Reply:
```jsx
interface ChipProps extends HTMLAttributes<HTMLDivElemtents> {
  label: string;
}
```

이렇게 만드니까 Element에 있는 속성에 대한 타입을 일일이 명시안해줘도 돼서 괜찮다..!!

---

> 참고로 난 on과 handle의 구분을 다음과 같이 하는데 이 네이밍은 같이 정해보자!
> 
> on : DOM 노드와 유사한 사용성을 위해 prop들 이름으로 사용
> handle: 컴포넌트 내부에서 커스텀 함수를 네이밍 할 때 사용

이거는 노션에 컨벤션 룰에 소희가 말한대로 작성되어 있어~ 세은 확인해주세요~
https://www.notion.so/dduckyee/8b576474c8cb4e0c90c7a709cc1f25de#859ed8515f69476a8905107a8dea6e21

Reply:
내가 만들었는데 적용 안했네 ...네이밍도 고칠게요~